src/NumPyDType.chpl

```diff
@@ -14,6 +14,7 @@ module NumPyDType
     */
     proc whichDtype(type etype) param : DType {
       if (etype == int) {return DType.Int64;}
+      if (etype == uint(64)) {return DType.UInt64;}
```

Comment:
I think the default size is 64 for both int and uint. we don't carry the `(64)` around in the code because it is the default. @bradcray what do you think?

Reply:
I agree that `uint(64)` is unnecessary and that `uint` should suffice.  Even if/when we add `uint(128)` or `int(128)` in the future, I don't think we'll be redefining `int` or `uint` or making it "implementation-defined" as C does.  

I can imagine different codes choosing to use either notation from a "completely self-documenting" vs. "brevity" angle.  In cases like this, I'd probably just follow the lead of the existing code and use `uint` ("if it was good enough for them, it's good enough for us").